src/main/java/org/opendcs/odcsapi/dao/ApiComputationDAO.java

```diff
@@ -522,7 +522,7 @@ else if (name.toLowerCase().endsWith("_missing"))
 	}
 		
 	/**
-	 * Create computation object & fill with current row of result set.
+	 * Create computation object and fill with current row of result set.
```

Comment:
alternatively `&amp;` (for javadoc)

Reply:
yeah, I just didn't find a case where that would be nicer to see than "and"